web/components/roomConfig/RoomConfig.tsx

```diff
@@ -86,7 +93,7 @@ export const RoomConfig = function<SConfig, ConnectionType extends GetConnection
 
     return <main>
         {
-            error && <ErrorPane header={error.header || "Error"}>{error.message}</ErrorPane>
+            error && <ErrorPane header={error.header || "Error"} isWarning={error.isWarning}>{error.message}</ErrorPane>
```

Comment:
I'd either do <ErrorPane kind="warning"> *or* <WarningPane> (preference on the latter). `isWarning` feels clunky.